src/WebJobs.Script/ExtensionBundle/ExtensionBundleManager.cs

```diff
@@ -308,8 +308,11 @@ private NuGetVersion GetStandardOrExtendedBundleVersion(IList<NuGetVersion> orde
         {
             if (orderedByDescBundlesList.Count > 1)
             {
+                var previous = orderedByDescBundlesList[1];
+                _logger.LogInformation("Applying platform release channel configuration {platformReleaseChannelName}. Bundle version {previous} will be used", _platformReleaseChannel, previous);
```

Comment:
Can we change all of these logsto debug or trace? Why do they need to be info

Reply:
Here's the levels we currently use for the different extensions bundles logs (information and higher): https://github.com/Azure/azure-functions-host/blob/dev/src/WebJobs.Script/Diagnostics/Extensions/ExtensionBundleLoggerExtension.cs. 

It logs non-fatal errors as Information as well (we have some non-fatal error scenarios covered in this PR's logs) and some additional things like the path the bundles were found at. These logs are also only logged on startup when extension bundles are loaded (so they won't be excessively noisy). 

I think we could lower the level for non-error logs that I'm adding here but wanted to note the existing logging behavior of this class.

Reply:
To @liliankasem's point though, we have an over-usage of `LogInformation`. We should start considering lower verbosity for many of our logs.

Reply:
Results of offline discussion:

Keeping the logs informational in this case after tightening the scenarios/cases where they are logged. Bumped the non-fatal errors to warning.